### ISSUE TITLE
[Bug][SplitButton] Misleading message when using JAWS Screen reader #2620

### ISSUE THREAD

**Issue by filipKovachev** [telerik/kendo-react#1634]
When using JAWS Screen Reader on a focused SplitButton the following message is read:

"button menu, press space to activate the menu then navigate with arrowkey"

This is misleading because pressing space does not open the menu and it cannot be navigated via the arrow keys this way, but by pressing Alt+Down Arrow.

This can be reproduced here:

https://stackblitz.com/edit/react-seqtjy?file=app%2Fmain.tsx

Ticket ID: 1614886
